src/main/webapp/filter.js

```diff
@@ -62,8 +62,17 @@ function sendClickedFiltersToServer() { //eslint-disable-line
     DIFF_ARRAY_URL_PARAM + '=' + jsonDiffArray,
   {method: 'POST'}).then((response) => response.text())
       .then((message) => {
-        // TODO: take out, replace with success or error message
-        console.log(message);
+        // If success, redirect to scavenger hunt with ID
+        // If failure, display on screen
+        if (message != 'Error\n') {
```

Comment:
Could you use trim here?
https://www.w3schools.com/jsref/jsref_trim_string.asp

Reply:
It might also be better to return a json object with a success boolean and a hunt_id field if it was successful, but this doesn't seem to be causing any issues at the moment, so it's okay